wca-guidelines.md

```diff
@@ -222,6 +222,8 @@ To be more informative, each Guideline is classified using one of the following
 
 ## <article-H><multiple-blindfolded><multipleblindfoldedsolving> Article H: Multiple Blindfolded Solving
 
+- H1a+) [ADDITION] The competitors may choose to bring 1 or 2 extra puzzles as backup in case of incidents found during the attempt (e.g. duplicate scrambled puzzles). Attempts higher than 6 puzzles are highly recommendated to bring extra puzzles.
```

Comment:
Another language issues is see here: in the last sentence, "attempts" cannot bring extra puzzles.
Maybe "For attempts with more than 6 puzzles, extra puzzles are highly recommended." or "Competitors attempting more than 6 puzzles should bring extra puzzles."